packages/jsii-pacmak/lib/targets/go/package.ts

```diff
@@ -469,14 +469,6 @@ export class RootPackage extends Package {
     );
   }
 
-  protected emitHeader(code: CodeMaker) {
-    if (this.assembly.description !== '') {
-      code.line(`// ${this.assembly.description}`);
-    }
-    code.line(`package ${this.packageName}`);
-    code.line();
-  }
```

Comment:
I think this should still be emitted.... but _only_ in a single file (for example, `main.go`)

Reply:
I see, will make the update. Thank you.